docs/Antora/modules/howto_guides/pages/howto_configure_streaming.adoc

```diff
@@ -0,0 +1,298 @@
+
+    // Find and connect to a device hosting an OPC UA TMS server
+    const auto availableDevices = instance.getAvailableDevices();
+    daq::DevicePtr device;
+    for (const auto& deviceInfo : availableDevices)
+    {
+        if (deviceInfo.getConnectionString().toView().find("daq.opcua://") != std::string::npos)
+        {
+            // Add device using modified configuration Property object
+            device = instance.addDevice(deviceInfo.getConnectionString(), deviceConfig);
+            break;
+        }
+    }
+
+    if (!device.assigned())
+        std::cerr << "No relevant device found!" << std::endl;
+    else
+        // Output the name of the added device
+        std::cout << device.getInfo().getName() << std::endl;
+    
+    return 0;
+}
+----
+====
+
+[#streaming_config_per_signal]
+== Configure Streaming per Signal
+
+Once the xref:glossary:glossary.adoc#device[Device] is connected, the Streaming sources of its
+xref:glossary:glossary.adoc#signal[Signal]s can be examined and modified for each signal individually
+at any given time.
+
+The Streaming sources are identified by a connection string that includes the protocol prefix, indicating
+the protocol type ID, and parameters based on the protocol type (IP address, port number etc.).
+To manipulate the Streaming sources of particular Signal the `MirroredSignalConfig` object is used,
+it provides ability to:
+
+* retrieve a list of streaming sources available for signal,
+* get the currently active streaming source,
+* modify the active source or unset it to disable data streaming for a signal.
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+
+using namespace daq;
+
+// Corresponding document: Antora/modules/howto_guides/pages/howto_configure_streaming.adoc
+int main()
+{
+    ...
+
+    // Get the first signal of conencted device
+    daq::SignalPtr signal = device.getSignalsRecursive()[0];
+
+    daq::StringPtr nativeStreamingSource;
+    daq::StringPtr websocketStreamingSource;
+
+    // Find and output the streaming sources available for signal
+    std::cout << "Signal supports " << signal.getStreamingSources().getCount()
+              << " streaming sources:" << std::endl;
+    for (const auto& source : signal.getStreamingSources())
+    {
+        std::cout << source << std::endl;
+        if (source.toView().find("daq.ns://") != std::string::npos)
+            nativeStreamingSource = source;
+        if (source.toView().find("daq.wss://") != std::string::npos)
+            websocketStreamingSource = source;
+    }
+
+    // Output the active streaming source of signal
+    std::cout << "Active streaming source of signal: " << signal.getActiveStreamingSource();
+
+    // Change the active streaming source of signal
+    signal.setActiveStreamingSource(nativeStreamingSource);
+
+    std::cout << "Press \"enter\" to exit the application..." << std::endl;
+    std::cin.get();
+    return 0;
+}
+----
+====
+
+== Full listing
```

Comment:
Maybe a mention of pseudo devices would be useful somewhere in this how-to.

Reply:
"Connecting to Streaming protocol based pseudo-devices" how-to section added